interactions/api/models/channel.py

```diff
@@ -508,6 +510,274 @@ async def get_pinned_messages(self):
         messages = [Message(**message, _client=self._client) for message in res]
         return messages
 
+    async def get_message(
+        self,
+        message_id: int,
+    ):
+        """
+        Gets a message sent in that channel.
+
+        :return: The message as object
+        :rtype: Message
+        """
+        res = await self._client.get_message(
+            channel_id=int(self.id),
+            message_id=message_id,
+        )
+        from .message import Message
+
+        return Message(**res, _client=self._client)
+
+    async def purge(
+        self,
+        amount: int,
+        check: Callable = MISSING,
+        before: Optional[int] = MISSING,
+        reason: Optional[str] = None,
+        bulk: Optional[bool] = True,
+    ) -> List:
+        """
+        Purges a given amount of messages from a channel. You can specify a check function to exclude specific messages.
+        .. code-block:: python
+            def check_pinned(message):
+                return not message.pinned  # This returns `True` only if the message is the message is not pinned
+            await channel.purge(100, check=check_pinned)  # This will delete the newest 100 messages that are not pinned in that channel
+        :param amount: The amount of messages to delete
+        :type amount: int
+        :param check?: The function used to check if a message should be deleted. The message is only deleted if the check returns `True`
+        :type check: Callable[[Message], bool]
+        :param before?: An id of a message to purge only messages before that message
+        :type before: Optional[int]
+        :param bulk?: Whether to bulk delete the messages (you cannot delete messages older than 14 days, default) or to delete every message seperately
+        :param bulk: Optional[bool]
+        :param reason?: The reason of the deletes
+        :type reason: Optional[str]
+        :return: A list of the deleted messages
+        :rtype: List[Message]
+        """
+        if not self._client:
+            raise AttributeError("HTTPClient not found!")
+        from .message import Message
+
+        _before = None if before is MISSING else before
+        _all = []
+        if bulk:
+            _allowed_time = pytz.UTC.localize(datetime.now() - timedelta(days=14))
```

Comment:
Why are we using `pytz` as a new dependency for handling time? Is there not a datetime equivocation?